Update documentation for coding style and camelCase usage

jNullj opened the pull request:
Clarify coding style guidelines in CONTRIBUTING.md and provide specific examples of camelCase usage in TUTORIAL.md and badge-urls.md.

Fixes #10846

cyb3rko reviewed (commented):
Maybe explicity state using "lower camel case", as there is lower and upper/capital camel case.  
(https://en.wikipedia.org/wiki/Camel_case)

jNullj commented:
@cyb3rko i added an explicit statement that camelCase is lower camelCase. Thank you for the review.
@PyvesB I think we should, updated to latest commit.
